### ISSUE TITLE
Reminder: Selection of Submissions for Final Ranking 

### ISSUE THREAD

**Issue by aisg-py** [AISG-Technology-Team/GCSS-Track-1B-Submission-Guide#25]
Hi all,
As stated on the challenge portal, you must select up to two submissions for the final private leaderboard ranking before the Track 1B submission deadline. Failure to make a selection will result in exclusion from the ranking.

Thank you.

**Comment by nonstopfor** [comment]
Can we include a running submission for the final private leaderboard ranking, if the submission doesn't finish before the submission deadline?

**Comment by aisg-py** [comment]
No, you cannot include a submission that is still running for the final private leaderboard ranking if it does not finish before the submission deadline. The submission must have completed running to be eligible for inclusion.

**Comment by nonstopfor** [comment]
It seems we can't select a running submission for the final private leaderboard ranking on the website?

**Comment by CeleneDion** [comment]
Same question. There is no option to select a running submission on the website.

**Comment by aisg-py** [comment]
Apologies, we have updated our response.

**Comment by CeleneDion** [comment]
Our last submission is almost done. Can you let it finish and cout it as our final submission?

**Comment by aisg-py** [comment]
To ensure fairness for all participants, we will adhere to the submission deadline as stipulated on the challenge website.
